repo/packages/H/helloworld/0/marathon.json

```diff
@@ -3,7 +3,7 @@
   "cpus": 1.0,
   "mem": 512,
   "instances": 1,
-  "cmd": "python3 -m http.server 8080",
+  "cmd": "python3 -m http.server 8487",
```

Comment:
Let's make this dynamic. Take a look at https://github.com/mesosphere/universe/blob/master/repo/packages/M/marathon/0/marathon.json#L7-L13

You can use it here as:
```
"cmd": "python3 -m http.server $PORT0",
```